OpenRA.Game/Game.cs

```diff
@@ -463,9 +463,12 @@ public static void InitializeMod(string mod, Arguments args)
 			Renderer.InitializeDepthBuffer(grid);
 
 			Cursor?.Dispose();
-
 			Cursor = new CursorManager(ModData.CursorProvider);
 
+			var metadata = ModData.Manifest.Metadata;
+			if (!string.IsNullOrEmpty(metadata.ApplicationTitle))
+				Renderer.Window.SetApplicationTitle(metadata.ApplicationTitle);
```

Comment:
How about falling back to `$"{metadata.Title - OpenRA}"`, so that we then don't need to change anything in the default mod.yamls?

Reply:
In any case, we should use "OpenRA - {MOD}" instead of "{MOD} - OpenRA" to be consistent with the OS desktop integration, windows start menu, etc.

Reply:
Look at your browser: it doesn't say "Firefox - OpenRA.net" but "OpenRA.net - Firefox" so it is still informational when you have many tabs open in your task bar. I always hate it when applications deviate from that standard, so I won't change it. I also don't see the need to hard-code even more pre-configuration. This also acts as a documentation for the feature.

Reply:
My point was that this PR is deviating from our standard...

Windows:
<img width="292" alt="Screenshot 2021-07-17 at 14 48 48" src="https://user-images.githubusercontent.com/167819/126039116-b72204aa-d090-4b1b-b330-b36a7d2f6703.png">

macOS:
<img width="960" alt="Screenshot 2021-07-17 at 14 46 45" src="https://user-images.githubusercontent.com/167819/126039122-8bd5c7c5-7449-4c5f-bc71-ec8efdb06ce7.png">

Linux:
<img width="579" alt="Screenshot 2021-07-17 at 14 52 31" src="https://user-images.githubusercontent.com/167819/126039203-ec38ecc6-4bad-4d63-ab9b-5de77cd8a6ff.png">

Reply:
And you already see that standard is not a good idea, because the mod title is truncated.

Reply:
I agree with @Mailaender here - "{MOD} - OpenRA" is better than "OpenRA - {MOD}" because the important information is at the front and will be visible when the UI can't fit the full title. I can see a benefit of putting OpenRA at the front for keeping the items together when sorting alphabetically but that's about it :thinking: And IMO the packaging should also change to come in line with this new format.

Reply:
I would agree if the behaviour when truncated were the only motivating factor.

However, we need to consider the implications of how we present our "brand". I believe that our current naming convention is consistent with the [franchise modding guidelines](https://www.ea.com/games/command-and-conquer/command-and-conquer-remastered/modding-faq) because we present them primarily as being OpenRA, with the game title as a descriptive addon that I consider to fall under "you may state that your Mod is for Command & Conquer and/or for a specific C&C game or expansion pack in the franchise".

I personally feel that we would be violating the spirit of this agreement by deemphasising the OpenRA and presenting them as Red Alert, Tiberian Dawn etc. The fact that they are OpenRA projects *is* the important information, so it is important for that to stay at the front.

Reply:
Changed according to your brand guidelines.